atc/plan.go

```diff
@@ -28,7 +28,8 @@ type Plan struct {
 type PlanID string
 
 type UserArtifactPlan struct {
-	Name string `json:"name"`
+	ArtifactID int    `json:"artifact_id"`
+	Name       string `json:"name"`
 }
 
 type ArtifactOutputPlan struct {
```

Comment:
Should this be removed?

Reply:
nah, we still need it

Reply:
err I originally thought you we're talking about the `UserArtifactPlan` because of how its highlighted. I now realize you actually meant the ArtifactOutputPlan, which coincidently we are bringing back because of other things you mentioned :)

Reply:
glad it never got removed in the first place ;)

Reply:
lol, yep 👍